java/gast_to_code/gast_to_code_java.py

```diff
@@ -149,3 +165,29 @@ def handle_assign_pattern(gast):
 
     def handle_arr(gast):
         return "{" + router.gast_to_code(gast["elements"], "java") + "}"
+
+    def handle_root(gast):
```

Comment:
Also could use some comments since this is not very intuitively clear. Feel like if I were to come back in a month to look at this I would be confused even though i helped write it.

Reply:
+1